packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -7,7 +7,7 @@ export const getMostRecentReadReceipts = createSelector(
   [getActivities, getParticipants],
   (activities, participants) => {
     const activity = activities.last();
-    return participants.filter((participant) =>
+    return participants.toJS().filter((participant) =>
```

Comment:
I could've done ```participant.get(`roomProperties`)``` within the filter, but then we would still need to do the `toJS` method since the read-receipts component is expecting a basic javascript object.